reporter/otlp_reporter.go

```diff
@@ -86,8 +86,10 @@ type traceEvents struct {
 
 // attrKeyValue is a helper to populate Profile.attribute_table.
 type attrKeyValue[T string | int64] struct {
-	key   string
-	value T
+	key string
+	// Set to true for OTel SemConv attributes with requirement level: Required
+	required bool
```

Comment:
Can you link to required attributes that we currently send?

Reply:
According to OTel SemConv there is no attribute marked as `Required` currently.